satellite/audit/reservoir.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Paths   []storj.Path
+	Size    int
+	NumSeen int
+}
+
+// NewReservoir instantiates a Reservoir
+func NewReservoir(size int) *Reservoir {
+	return &Reservoir{
+		Size:    size,
+		NumSeen: 0,
+	}
+}
+
+// sample makes sure that for every segment in metainfo from index i=numSlots..n-1,
```

Comment:
Use `size` instead of `numSlots` in this comment, so it matches the reservoir's field.